suites/compaction/deployments/rbd-mirror-deploy-and-configure.yaml

```diff
@@ -0,0 +1,189 @@
+tests:
```

Comment:
please mention which conf to use to run this test suite

Reply:
done